presentation-api/index.html

```diff
@@ -29,6 +29,10 @@
   <button id="reconnect" disabled>presentationRequest.reconnect(presentationId)</button>
 </p>
 <p>
+  <input id="message" type="text" placeholder="Try a fruit!">
```

Comment:
Please use a datalist that contains all fruits we handle so that it is easier for user to play with:
```html
<input id="message" type="text" placeholder="Try a fruit!" list="fruits">

<datalist id="fruits">
  <option>grapes</option>
  ...
</datalist>